examples/get-config-from-file/index.ts

```diff
@@ -0,0 +1,29 @@
+import { FileHelpers, TypeScriptGenerator, TypeScriptOptions } from '../../src';
+
+const jsonSchemaDraft7 = {
+  $schema: 'http://json-schema.org/draft-07/schema#',
+  type: 'object',
+  additionalProperties: false,
+  properties: {
+    email: {
+      type: 'string',
+      format: 'email'
+    }
+  }
+};
+
+async function main() {
+  try {
+    const config: { config: TypeScriptOptions } =
+      await FileHelpers.getConfigFromFile('./modelina.config.alternate.js');
```

Comment:
I think we should simplify this.

```
    const config = await FileHelpers.getConfigFromFile('./modelina.config.alternate.js');
```

Instead of having to do something like `config.config`, what do you think? 🤔

Reply:
Yeah definitely, it looks absurd.
We can directly export the config. :slightly_smiling_face: 
Would that work ?